### DISCUSSION TITLE
New script request: Home Assistant Assist tools

### DISCUSSION THREAD

**Discussion by titiviking** [community-scripts/ProxmoxVE#853] [Request script]
Can there be a script to install LXC with GPU support for:
- Open Wake word
- Piper
- Whisper

(all should support wyoming protocol for easy integration into HA).

Many thanks, looking forward to it!

**Comment by havardthom** [comment]
Please use the template for script requests: https://github.com/community-scripts/ProxmoxVE/discussions/new?category=request-script
